programs/programs/urgent_needs/tx/trust_her.py

```
@@ -5,16 +5,24 @@ class TrustHer(UrgentNeedFunction):
-    support for Medicaid and coverage programs.
+    Free or low-cost birth control and reproductive health services for Dallas women
+    and teens, with same-day access to all contraception methods.
 
-    Dallas County; women and teens; no age minimum; free or low cost based on income.
-    County restriction managed via admin configuration.
+    Dallas County ZIP code (county restriction managed via admin); 250% FPL or less;
+    at least one household member has no health insurance.
     """
 
-    dependencies = []
+    dependencies = ["income_amount", "income_frequency", "household_size", "health_insurance"]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)